molten-cache/src/test/java/com/hotels/molten/cache/ReactiveMapCacheTest.java

```diff
@@ -65,4 +68,46 @@ public void shouldDelegatePutLazily() {
             .then(() -> verify(cache).put(KEY, VALUE))
             .verifyComplete();
     }
+
+    @Test
```

Comment:
These default method could have their own test class as they affect all implementation.

Reply:
Created a contract test class. I think all the cache implementations' test class could extend that eventually. But that might be part of another PR.